src/main/java/edu/ucsb/cs156/courses/jobs/UpdateCourseDataJob.java

```diff
@@ -22,6 +23,26 @@ public class UpdateCourseDataJob implements JobContextConsumer {
     @Getter private String quarterYYYYQ;
     @Getter private UCSBCurriculumService ucsbCurriculumService;
     @Getter private ConvertedSectionCollection convertedSectionCollection;
+    @Getter private UpdateCollection updateCollection;
+
+    // public void setLastUpdateTime(UpdateCollection uc, String subjectArea, String quarterYYYYQ) {
+    //     Optional<Update> optionalUpdate =
+    //         uc.findOneBySubjectAreaAndQuarterYYYYQ(subjectArea, quarterYYYYQ);
+
+    //     if (optionalUpdate.isPresent()) {
+    //         Update existingUpdate = optionalUpdate.get();
+    //         existingUpdate.setLastUpdate(ZonedDateTime.now());
+    //         uc.save(existingUpdate);
+    //     } else {
+    //         uc.save(
+    //             Update.builder()
+    //                 .subjectArea(subjectArea)
+    //                 .quarterYYYYQ(quarterYYYYQ)
+    //                 .lastUpdate(ZonedDateTime.now())
+    //                 .build()
+    //         );
+    //     }
+    // }
```

Comment:
We try not to put commented out code into PRs to main.

Please remove this.

Reply:
OK! I will remove this. I was not planning on this commit being merged. I cannot run the pitest locally for some reason, and I just wanted to make sure that this was what was giving me errors.